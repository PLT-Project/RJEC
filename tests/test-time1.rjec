func main()
{
    i := time();
    prints("time elapsed greater than 0:");
    printb(i < time());
}