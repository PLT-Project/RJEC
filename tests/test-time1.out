time elapsed greater than 0:
true